invenio_github/models.py

```diff
@@ -161,18 +161,35 @@ class Repository(db.Model, Timestamp):
     hook = db.Column(db.Integer)
     """Hook identifier."""
 
+    #
+    # Relationships
+    #
     user = db.relationship(User)
 
     @classmethod
-    def get(cls, user_id, github_id=None, name=None, create=False):
+    def get_or_create(cls, user_id, github_id=None, name=None, **kwargs):
```

Comment:
Maybe it was originally bad idea to do two things in one functions. What about keeping low-level API clean with just get and create methods?